MOxUnit/util/moxunit_util_escape_xml.m

```diff
@@ -0,0 +1,33 @@
+function str_esc=moxunit_util_escape_xml(str)
```

Comment:
I cloned your repo and ran your test, and get the following xml output:

    <testcase classname="my_test_of_abs" name="test_abs_error" time="0.005">
      <failure message="Error using my_test_of_abs&gt;test_abs_error (line 19)
    elements are not equal
    
    First input:
    7
    
    Second input:
    8
    ">  test_abs_error:19 (/Users/nick/organized/tmp/matlab-ci/tests_moxunit/my_test_of_abs.m)
      run:24 (/Users/nick/git/MOxUnit/MOxUnit/@MOxUnitFunctionHandleTestCase/run.m)
      run:25 (/Users/nick/git/MOxUnit/MOxUnit/@MOxUnitTestSuite/run.m)
      run:25 (/Users/nick/git/MOxUnit/MOxUnit/@MOxUnitTestSuite/run.m)
      moxunit_runtests:78 (/Users/nick/git/MOxUnit/MOxUnit/moxunit_runtests.m)
      runAllTests:19 (/Users/nick/organized/tmp/matlab-ci/runAllTests.m)</failure>
    </testcase>

That looks fine, doesn't it?